tests/cypress/fixtures/orderingList.graphql

```diff
@@ -0,0 +1,12 @@
+query($nodeType:String!, $orderType:OrderType, $property:String){
+    jcr(workspace:EDIT){
+        nodesByCriteria(criteria: {paths: "/testList", nodeType: $nodeType, ordering: {orderType:$orderType, property:$property}})
```

Comment:
Although not a requirement (since `sortingList.graphql` does not do it neither), but one suggestion here could be to use a variable for the path.